rotkehlchen/tasks/calendar.py

```diff
@@ -0,0 +1,43 @@
+from typing import NamedTuple
+
+from rotkehlchen.api.websockets.typedefs import WSMessageType
+from rotkehlchen.db.calendar import CalendarEntry
+from rotkehlchen.db.dbhandler import DBHandler
+from rotkehlchen.user_messages import MessagesAggregator
+
+
+class CalendarNotification(NamedTuple):
+    """Basic reminder information along the calendar event linked to it"""
+    event: CalendarEntry
```

Comment:
Isn't this copying the structure from yesterday's PR? Just re-use it. Why duplicate it here?

Reply:
is not the same. the other used only the event id and this one has all the information about the event since the frontend doesn't keep it in memory

Reply:
So can't you inherit yesterday's one? Since it just has one extra attribute? Why duplicate?

Reply:
It isn't an extra attribute. The other one uses event_id and this one uses the whole calendar event. I can abstract the two fields but I'm not sure if it hurts that much